### DISCUSSION TITLE
以deb包格式分发

### DISCUSSION THREAD

**Discussion by rong-xiaoli** [qier222/YesPlayMusic#2133] [Ideas]
（个人极其喜欢deb包格式安装）
所以想问一下可不可以有deb格式的安装包出来，谢谢
以及如果不会的话，可以问以下原因吗

**Comment by StLeoX** [comment]
可以对 Appimage 格式解压缩，和 deb 解压缩没什么差别。
https://superuser.com/questions/1301583/how-can-i-extract-files-from-an-appimage
